stob/linux.c

```diff
@@ -267,7 +267,6 @@ static int stob_linux_domain_init(struct m0_stob_type *type,
 static void stob_linux_domain_fini(struct m0_stob_domain *dom)
 {
 	struct m0_stob_linux_domain *ldom = m0_stob_linux_domain_container(dom);
-
```

Comment:
Blank line between declarations and code is okay.

Reply:
incorporated